gulpfile.js

```diff
@@ -66,7 +67,7 @@ gulp.task('watch', function () {
   gulp.watch([config.basepath.src + '/assets/includes/**/*.html'], ['includes']);
   gulp.watch([config.basepath.src + '/**/*.scss'], ['scss']);
   gulp.watch([config.basepath.src + '/**/*.js'], ['js']);
-  gulp.watch([config.basepath.src + '/**/*.js'], ['eslint']);
+  gulp.watch([config.basepath.src + '/**/*.js'], ['test:eslint:soft']);
```

Comment:
It would be good if we have inline documentation about what each task is doing.
Ex: 'includes'->moves /src/assets/includes folder